tpp-app/src/home/pages/LogSymptomsScreen.js

```diff
@@ -202,7 +204,12 @@ export default function LogSymptomsScreen({ navigation, route }) {
 
     POSTsymptomsForDate(selectedDate.getDate(), selectedDate.getMonth() + 1, selectedDate.getFullYear(), submitSymp)
       .then(() => {
-        navigation.goBack();
+        let inputData = {}
```

Comment:
I would make this into a function because you need to do this everywhere it navigates back to STACK_SCREENS.CALENDAR from both LogSymptomsScreen.js and LogMultipleDatesScreen.js